code/datums/autolathe/biomatter.dm

```diff
@@ -189,6 +193,7 @@
 
 /datum/design/bioprinter/nt_clothes/knight_rig
 	name = "Knight Hardsuit Module"
+	materials = list(MATERIAL_BIOMATTER = 50, MATERIAL_GOLD = 5, MATERIAL_SILVER = 5, MATERIAL_PLASTEEL = 10)
```

Comment:
**VERY** appreciated, I got complains from church players that the armor would've been _"too good"_ for its values and utility, and it only took metal and plastic to build, this selection of materials is pretty fair and balanced.